src/write_gcm.c

```diff
@@ -2836,6 +2848,10 @@ static int wg_json_CreateCollectdTimeseriesRequest(_Bool pretty,
   wg_json_string(jc, "name");
   wg_json_string(jc, name);
 
+  // Request partial errors from the server, so we can log them.
+  wg_json_string(jc, "return_partial_errors");
```

Comment:
The Logging API calls this [`partialSuccess`](https://cloud.google.com/logging/docs/reference/v2/rest/v2/entries/write#request-body). Any reason to diverge here?